lms/djangoapps/courseware/views/views.py

```diff
@@ -1467,6 +1468,15 @@ def render_xblock(request, usage_key_string, check_if_enrolled=True):
         student_view_context = request.GET.dict()
         student_view_context['show_bookmark_button'] = False
 
+        enable_completion_on_view_service = 'false'
```

Comment:
Why using string to specify Boolean values?
If the values here are meant for the javascript, can we use Boolean to string `str()`?

Reply:
I have changed this to a Boolean.